manual/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Creates and installs manual page
+
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -E create_symlink ${CMAKE_CURRENT_SOURCE_DIR}/nominatim.1 ${CMAKE_CURRENT_BINARY_DIR}/nominatim.1
```

Comment:
You can't link here because then the install command below will later only copy the symbolic link instead of installing the actual file.

I would suggest to simply always work with the manpage in the source directory: install that one directly and when regenerating with 'make manpage' put that into the source directory directly.

Reply:
Ah, Sorry, I didn't knew that. I will make changes.